app/assets/stylesheets/components/blank_slate.scss

```diff
@@ -23,6 +23,12 @@
         height: 0;
         padding-bottom: 144%;
       }
+      &.alec {
+        background-image: image-url("illustrations/contributors/alec-talking.png");
+      }
+      &.jenya {
+        background-image: image-url("illustrations/contributors/jenya-talking.png");
+      }
```

Comment:
@ofsound Just a heads up that I'm going to add all the blank slate images here, so they are usable on all pages.